config/global.go

```diff
@@ -26,9 +26,11 @@ type Global struct {
 	LegacyArguments      bool          `mapstructure:"legacy-arguments" default:"false" deprecated:"0.20.0" description:"Legacy, broken arguments mode of resticprofile before version 0.15"`
 	SystemdUnitTemplate  string        `mapstructure:"systemd-unit-template" default:"" description:"File containing the go template to generate a systemd unit - see https://creativeprojects.github.io/resticprofile/schedules/systemd/"`
 	SystemdTimerTemplate string        `mapstructure:"systemd-timer-template" default:"" description:"File containing the go template to generate a systemd timer - see https://creativeprojects.github.io/resticprofile/schedules/systemd/"`
+	SystemdDropInFiles   []string      `mapstructure:"systemd-drop-in-files" default:"" description:"Files containing systemd drop-in (override) files - see https://creativeprojects.github.io/resticprofile/schedules/systemd/"`
```

Comment:
Wasn't sure whether this should be in `global` or the profiles (e.g. `default`), thinking about it maybe having it in profiles would be more useful

Reply:
Profiles is likely better. Since drop-ins are no templates and may be profile specific there would be no way to achieve it when it is a global setting.